docs/modules/operation/pages/events/sources/snmp-traps.adoc

```diff
@@ -2,23 +2,352 @@
+** <<varbind-filter,Filter on variable bindings>> for further granularity on the conditions for creating an event
+
+[[trap-def-create]]
+=== Create event definition for an SNMP trap
+{page-component-title} includes event definitions for traps from many vendors' equipment.
+Use the `mask` tag to match SNMP traps in `eventconf.xml`.
+See xref:operation:events/anatomy-events.adoc#ga-events-anatomy-of-an-event[Anatomy of an Event] for details on event structure.
+
+The following example shows a Cisco Systems event for their C3800 device.
+Parts of it look similar to internally generated events, with the main difference being the `mask` block.
+This block consists of <maskelement> tags, and the event will match only if all of the defined tags are met.
+
+.Cisco systems event
+[source, xml]
+----
+<event>
+  <mask>
+    <maskelement>
+      <mename>id</mename>
+      <mevalue>.1.3.6.1.4.1.9.9.70.2</mevalue>
+    </maskelement>
+    <maskelement>
+      <mename>generic</mename>
+      <mevalue>6</mevalue>
+    </maskelement>
+    <maskelement>
+      <mename>specific</mename>
+      <mevalue>17</mevalue>
+    </maskelement>
+  </mask>
+  <uei>http://uei.opennms.org/vendor/Cisco/traps/ciscoC3800SysAggregateStatusChange</uei>
+  <event-label>CISCO-C3800-MIB defined trap event: ciscoC3800SysAggregateStatusChange</event-label>
+  <descr>&#38lt;p&#38gt;Notification that the aggregate status of a node
+         has changed.&#38lt;/p&#38gt;&#38lt;table&#38gt;
+         &#38lt;tr&#38gt;&#38lt;td&#38gt;&#38lt;b&#38gt;
+         c3800SysNextTrapSeqNum&#38lt;/b&#38gt;</td&#38gt;&#38lt;td&#38gt;%parm[#1]%
+         &#38lt;/td&#38gt;&#38lt;td&#38gt;&#38lt;p;&#38gt;</p&#38gt;&#38lt;/td;&#38gt;&#38lt;/tr&#38gt;&#38lt;tr&#38gt;&#38lt;td&#38gt;&#38lt;b&#38gt;
+         sysName&#38lt;/b&#38gt;&#38lt;/td&#38gt;&#38lt;td&#38gt;%parm[#2]%
+         &#38lt;/td&#38gt;&#38lt;td&#38gt;&#38lt;p;&#38gt;&#38lt;/p&#38gt;&#38lt;/td;&#38gt;&#38lt;/tr&#38gt;&#38lt;tr&#38gt;&#38lt;td&#38gt;&#38lt;b&#38gt;
+         c3800SysTrapSeverity&#38lt;/b&#38gt;&#38lt;/td&#38gt;&#38lt;td&#38gt;%parm[#3]%
+         &#38lt;/td&#38gt;&#38lt;td&#38gt;&#38lt;p;&#38gt;
+         clear(1) minor(2) major(3)&#38lt;/p&#38gt;
+         &#38lt;/td;&#38gt;&#38lt;/tr&#38gt;&#38lt;tr&#38gt;&#38lt;td&#38gt;&#38lt;b&#38gt;
+         c3800SysAggregateStatus&#38lt;/b&#38gt;&#38lt;/td&#38gt;&#38lt;td&#38gt;%parm[#4]%
+         &#38lt;/td&#38gt;&#38lt;td&#38gt;&#38lt;p;&#38gt;
+         clear(1) minor(2) major(3)</p>
+         &#38lt;/td;&#38gt;&#38lt;/tr&#38gt;&#38lt;/table&#38gt;
+  </descr>
+  <logmsg dest='logndisplay'><p>Cisco Event: C3900: Node Status has changed.</p></logmsg>
+  <severity>Indeterminate</severity>
+</event>
+----
+
+This event will match an SNMP trap whose enterprise OID (id) is equal to `.1.3.6.1.4.1.9.9.70.2`, its generic trap value is enterprise specific `6`, and the vendor trap value is `17`.
```

Comment:
I'm not sure if this maintains the meaning we're going for here. If it doesn't, I would suggest breaking this into two sentences.

```suggestion
This event will match an SNMP trap where the enterprise OID (id) is equal to `.1.3.6.1.4.1.9.9.70.2`, the generic trap value is equal to `6` (enterprise-specific), and the vendor trap value is equal to `17`.
```

Reply:
It does maintain the meaning still. I read it just fine.